testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvSinglenodeOauth2.java

```diff
@@ -70,7 +70,7 @@ public void extendEnvironment(Environment.Builder builder)
                 "trinodb_client_id",
                 "trinodb_client_secret",
                 "client_secret_basic",
-                "trinodb_client_id/",
```

Comment:
If possible it's always nice to keep refactoring commits as first ones within PR.

Reply:
yes, I agree, however I thought the commit is not directly connected with the change ( as it can be added before or after without the influence on the actual change in PR) so I can leave as it is
on the other hand I can reshuffle